project_0a/project_0a_6.c

```diff
@@ -0,0 +1,40 @@
+/*
+Project 0a
+*/
+
+/*
+Exercise 5 - build a multiplication table for a given number. Use scanf() to
+read input from keyboard. Limit previous implementation (exercise 5) to not
+allow negative values nor positive higher than 50
+
+Note: if statement is represented as:
```

Comment:
It's not clear if this this a note left by the author or part of the exercise prompt, especially because the same notation was used in another file to express an author comment.